atlasdb-tests-shared/src/test/java/com/palantir/atlasdb/keyvalue/MemorySerializableTransactionTest.java

```diff
@@ -15,15 +15,47 @@
  */
 package com.palantir.atlasdb.keyvalue;
 
+import com.palantir.atlasdb.AtlasDbConstants;
 import com.palantir.atlasdb.keyvalue.impl.TestResourceManager;
 import com.palantir.atlasdb.transaction.impl.AbstractSerializableTransactionTest;
+import com.palantir.atlasdb.transaction.impl.TransactionConstants;
+import com.palantir.atlasdb.transaction.impl.TransactionSchemaVersionEnforcement;
+import com.palantir.timestamp.ManagedTimestampService;
+import java.util.Arrays;
+import java.util.Collection;
+import org.junit.Before;
 import org.junit.ClassRule;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
 
+@RunWith(Parameterized.class)
 public class MemorySerializableTransactionTest extends AbstractSerializableTransactionTest {
     @ClassRule
     public static final TestResourceManager TRM = TestResourceManager.inMemory();
 
-    public MemorySerializableTransactionTest() {
+    private static final String TRANSACTIONS_1 = "transactions1";
+    private static final String TRANSACTIONS_3 = "transactions3";
+
+    @Parameterized.Parameters(name = "{0}")
+    public static Collection<Object[]> data() {
+        Object[][] data = new Object[][] {
+            {TRANSACTIONS_1, TransactionConstants.DIRECT_ENCODING_TRANSACTIONS_SCHEMA_VERSION},
+            {TRANSACTIONS_3, TransactionConstants.TWO_STAGE_ENCODING_TRANSACTIONS_SCHEMA_VERSION}
```

Comment:
I think it's fine to just do 1 and 3, given we are looking to move off 2, and it was not tested before

Reply:
Agreed, that was my reasoning here